dockerfiles/remote-plugin-go-1.10.7/Dockerfile

```diff
@@ -42,17 +42,17 @@ RUN set -eux; \
 		/usr/local/go/pkg/obj \
 	; \
     export GOPATH="/go"; \
-    mkdir -p "$GOPATH/src" "$GOPATH/bin" && chmod -R 777 "$GOPATH"; \
+    mkdir -p "$GOPATH/src" "$GOPATH/bin" "$GOPATH/pkg" && chmod -R 777 "$GOPATH"; \
     export PATH="$GOPATH/bin:/usr/local/go/bin:$PATH"; \
     go get -u -v github.com/ramya-rao-a/go-outline && \
-    go get -u -v github.com/acroca/go-symbols &&  \ 
-    go get -u -v github.com/mdempsky/gocode && \ 
-    go get -u -v github.com/rogpeppe/godef && \ 
+    go get -u -v github.com/acroca/go-symbols &&  \
+    go get -u -v github.com/mdempsky/gocode && \
+    go get -u -v github.com/rogpeppe/godef && \
     go get -u -v golang.org/x/tools/cmd/godoc && \
     go get -u -v github.com/zmb3/gogetdoc && \
     go get -u -v golang.org/x/lint/golint && \
     go get -u -v github.com/fatih/gomodifytags &&  \
-    go get -u -v golang.org/x/tools/cmd/gorename && \
+    go get -u -v golang.org/x/tools/cmd/gorenam e && \
```

Comment:
typo ?

Reply:
@benoitf thank you! I did pay any attention to that :(